xcodeproj/internal/bazel_integration_files/import_indexstores.sh

```diff
@@ -100,6 +100,8 @@ remaps=(
   # these should always be Xcode.app relative, our regex supports command-line
   # tools based paths as well.
   -remap "^(?:.*?/[^/]+/Contents/Developer|/Library/Developer/CommandLineTools).*?/SDKs/([^\\d.]+)=$DEVELOPER_DIR/Platforms/\$1.platform/Developer/SDKs/\$1"
+
+  -remap "DEVELOPER_DIR=$DEVELOPER_DIR"
```

Comment:
How about this instead, so we only remap when `DEVELOPER_DIR` is at the start, and when it's for the sysroot?

```suggestion
  -remap "^(?:.*?/[^/]+/Contents/Developer|^DEVELOPER_DIR|/Library/Developer/CommandLineTools).*?/SDKs/([^\\d.]+)=$DEVELOPER_DIR/Platforms/\$1.platform/Developer/SDKs/\$1"
```

Reply:
I tested my suggestion and it works.

Reply:
Thanks @brentleyjones 🙏🏼